index.html

```diff
@@ -151,5 +151,23 @@ <h3>Skills</h3>
       </div>
       <p class="finalP">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
     </section>
+    <section class="Final-part">
+      <form action="https://formspree.io/f/mwkzgbqw" method="post" class="Contact-Form">
+        <input class="input1" type="text" id="name" name="user_name" placeholder="  Full name" required maxlength="30">
+        <input class="input1" type="email" id="email" name="user_email" placeholder="  Email Address" required>
+        <textarea class="input2" id="Message" name="user_message" required maxlength="500">  Enter text here</textarea>
```

Comment:
- [ ] The _**Enter text here**_ text for the `textarea` element should be a placeholder and not the default value for the form input. Kindly make it a placeholder 💻